api/tests/mocha/services/generate-xform.spec.js

```diff
@@ -326,6 +329,8 @@ describe('generate-xform service', () => {
         'form.html': { data: Buffer.from(currentForm) },
         'model.xml': { data: Buffer.from(currentModel) }
       } });
+
```

Comment:
In this file, I think you need to change all `db.medic.get` stubs to not return attachment data, because you changed the call parameters. 

So instead of: 
```
inon.stub(db.medic, 'get').resolves({ _attachments: {
        'xform.xml': { data: Buffer.from(formXml) },
        'form.html': { data: Buffer.from(currentForm) },
        'model.xml': { data: Buffer.from(currentModel) }
      } });
```

you'll have: 
```
sinon.stub(db.medic, 'get').resolves({ _attachments: {
        'xform.xml': { stub: true },
        'form.html': { stub: true },
        'model.xml': { stub: true }
      } });
```